### ISSUE TITLE
Feature request: A qualifier that detects the experiencer (beyond family members)

### ISSUE THREAD

**Issue by bdura** [aphp/edsnlp#68]
## Feature type

New qualifier pipeline, overtaking and improving the `eds.family` component.

## Description

As of today, EDS-NLP only equiped to detect family members. We should add detection of other types of experiencers, perhaps as a catch-all `other` modality?

Possible experiencers in that paradigm:
- `patient` or `self`
- `family`
- `other`

I feel this would cover most use-case. The `family` modality is particularly important in case of hereditary diseases (although one could argue we need to know whether those are blood relatives in that case). 

Depending on the speciality (psychology pops to mind), other types of experiencers might be useful (`friend`?), but I wonder whether the `other` experiencer is enough even in that case. The mere fact that the healthcare provider mentions an `other` could well be sufficient information?

Any thoughts?

**Comment by percevalw** [comment]
Good idea! I haven't read anything about extracting other types of experiencers than `self`, `family` and `other`, and I agree that these values cover most use cases.
As for detecting more specific experiencers such as friends, coworker, etc., we could rely on terminologies. For instance, `friend` is [C0079382](https://uts.nlm.nih.gov/uts/umls/concept/C0079382) and `coworker` is [C0681088](https://uts.nlm.nih.gov/uts/umls/concept/C0681088) in the UMLS.

**Comment by Thomzoy** [comment]
Another case of experiencer is the organ donor. 
Some notes go into great details to describe the donor's clinical state before death, leading to false positive when e.g. identifying diseases
